backend/src/mempool.interfaces.ts

```diff
@@ -320,6 +320,7 @@ export interface BlockExtension {
   segwitTotalSize: number;
   segwitTotalWeight: number;
   header: string;
+  firstSeen: string | null;
```

Comment:
I'd prefer `firstSeen` to be returned as a number, in unix seconds, for consistency with all of our other datetime fields (and to reduce ambiguity with timezones etc)